langchain4j-google-ai-gemini/src/test/java/dev/langchain4j/model/googleai/GoogleAiGeminiStreamingChatModelIT.java

```diff
@@ -617,7 +618,7 @@ void should_support_tool_config() {
             .apiKey(GOOGLE_AI_GEMINI_API_KEY)
             .modelName("gemini-1.5-flash")
             .logRequestsAndResponses(true)
-            .toolConfig(new GeminiFunctionCallingConfig(GeminiMode.ANY, List.of("toolTwo")))
+            .toolConfig(GeminiMode.ANY, List.of("toolTwo").toArray(new String[0]))
```

Comment:
this change looks strange, please explain

Reply:
in the function i was to accept the varargs (...String args) so i was just converting the list to an array of Strings 
i hae simplified it now